bazel/cc_wrapper.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python
+import os
+import sys
+import tempfile
+
+envoy_real_cc = {ENVOY_REAL_CC}
+envoy_real_cxx = {ENVOY_REAL_CXX}
+
+def sanitize_flagfile(in_path, out_fd):
+    with open(in_path, "rb") as in_fp:
```

Comment:
2 space indent, standard Google style for Python.

Reply:
Done with `yapf -i --style '{based_on_style: google, indent_width: 2}' cc_wrapper.py`